controllers/datafoundation.go

```diff
@@ -47,7 +47,10 @@ type dataFoundationReconciler struct {
 //+kubebuilder:rbac:groups=ocs.openshift.io,namespace=system,resources=storageclusters,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=network.openshift.io,resources=ingressnetworkpolicies,verbs=create;get;list;watch;update
 //+kubebuilder:rbac:groups=operators.coreos.com,namespace=system,resources=clusterserviceversions,verbs=get;list;watch;delete;update;patch
-//+kubebuilder:rbac:groups=ocs.openshift.io,namespace=system,resources=ocsinitializations,verbs=get;list;watch;update;patch
+//+kubebuilder:rbac:groups=ocs.openshift.io,namespace=system,resources=ocsinitializations,verbs=get;list;watch;update;patch;delete
+//+kubebuilder:rbac:groups=k8s.ovn.org,namespace=system,resources=egressfirewalls,verbs=create;get;list;watch;update
+//+kubebuilder:rbac:groups=network.openshift.io,namespace=system,resources=egressnetworkpolicies,verbs=create;get;list;watch;update
+//+kubebuilder:rbac:groups=apiextensions.k8s.io,resources=customresourcedefinitions,verbs=get;watch;list
```

Comment:
I see 4 tags here but only 3 rules were generated, maybe a duplicate was added 🤔 ?

Reply:
Updated the PR. One rule is getting updated.